modules/tracker/models/pdo/ScalarModel.php

```diff
@@ -145,12 +66,13 @@ public function delete($scalarDao)
      */
     public function getByTrendAndTimestamp($trendId, $submitTime, $userId = null)
     {
-        $sql = $this->database->select()->setIntegrityCheck(false)->where('trend_id = ?', $trendId)->where(
-            'submit_time = ?',
-            $submitTime
-        );
+        $sql = $this->database->select()->setIntegrityCheck(false)->from(array('s' => 'tracker_scalar'))->join(
```

Comment:
I mentioned in a comment on ScalarModelBase that maybe we shouldn't support this function anymore.

Looking at the implementation, it finds a scalar with the same submission_id and also the same submit_time.  This should be the same thing as just getting a scalar with the same submission_id.

Again, I'm not sure we need this anymore, and there is a bit of a disconnect between function name/docs and the implementation.